src/app/dim-ui/Dropdown.tsx

```diff
@@ -99,7 +99,8 @@ export default function Dropdown({
                     index,
                     onClick: !item.disabled
                       ? item.onSelected
-                      : (e: any) => {
+                      : // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Yeah I was testing with it on. These are here partly because I forgot and included them, but also because there's no good typing for these special `downshift` properties stuck onto events.